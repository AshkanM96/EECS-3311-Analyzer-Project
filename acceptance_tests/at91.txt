open_set_enumeration
boolean_constant(false)
conjoin
boolean_constant(false)
boolean_constant(true)
for_all
open_set_enumeration
boolean_constant(true)
boolean_constant(true)
conjoin
boolean_constant(false)
boolean_constant(true)
close_set_enumeration
close_set_enumeration 
type_check
evaluate 
