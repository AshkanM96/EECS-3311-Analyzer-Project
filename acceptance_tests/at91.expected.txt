  Expression currently specified: ?
  Report: Expression is initialized.
->open_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {False, ?}
  Report: OK.
->conjoin
  Expression currently specified: {False, (? && nil)}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {False, (False && ?)}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {False, (False && True), ?}
  Report: OK.
->for_all
  Expression currently specified: {False, (False && True), (&& ?)}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {False, (False && True), (&& {?})}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {False, (False && True), (&& {True, ?})}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {False, (False && True), (&& {True, True, ?})}
  Report: OK.
->conjoin
  Expression currently specified: {False, (False && True), (&& {True, True, (? && nil)})}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {False, (False && True), (&& {True, True, (False && ?)})}
  Report: OK.
->boolean_constant(True)
  Expression currently specified: {False, (False && True), (&& {True, True, (False && True), ?})}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {False, (False && True), (&& {True, True, (False && True)}), ?}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {False, (False && True), (&& {True, True, (False && True)})}
  Report: OK.
->type_check
  Expression currently specified: {False, (False && True), (&& {True, True, (False && True)})}
  Report: {False, (False && True), (&& {True, True, (False && True)})} is type-correct.
->evaluate
  Expression currently specified: {False, (False && True), (&& {True, True, (False && True)})}
  Report: {False}
